ee/tabby-webserver/src/service/page.rs

```diff
@@ -144,6 +144,9 @@ impl PageService for PageServiceImpl {
             let section = db
                 .create_page_section(page_id, &new_section_title)
                 .await?;
+            let section_id = section.id;
+            let page_section: PageSection = section.into();
+            yield Ok(SectionRunItem::PageSectionCreated(page_section));
```

Comment:
https://demo.tabbyml.com/files/github/TabbyML/tabby/-/blob/main/ee/tabby-schema/src/schema/page/types.rs?plain=1#L242

To confirm: is the order of yielding matching its definition in `types.rs` after this PR?

Reply:
yes